src/frontend/komponenter/Fagsak/TilkjentYtelse/EndreUtbetalingsperiodeSkjema.tsx

```diff
@@ -0,0 +1,204 @@
+import * as React from 'react';
+
+import styled from 'styled-components';
+
+import { Knapp, Flatknapp } from 'nav-frontend-knapper';
+import { SkjemaGruppe, Radio } from 'nav-frontend-skjema';
+import { Element, Normaltekst } from 'nav-frontend-typografi';
+
+import {
+    FamilieRadioGruppe,
+    FamilieDatovelger,
+    ISODateString,
+    FamilieTextarea,
+    FamilieReactSelect,
+} from '@navikt/familie-form-elements';
+
+import { useTidslinje } from '../../../context/TidslinjeContext';
+import { IBehandling } from '../../../typer/behandling';
+import SkjultLegend from '../../Felleskomponenter/SkjultLegend';
+
+const Rad = styled.div`
+    display: flex;
+    flex-direction: row;
+`;
+
+const StyledSkjemaGruppe = styled(SkjemaGruppe)`
+    margin-top: 5rem;
+    padding-left: 3.75rem;
+    margin-right: 2rem;
+    border-left: 0.0625rem solid black;
+`;
+
+const StyledPersonvelger = styled(FamilieReactSelect)`
+    max-width: 20rem;
+`;
+
+const Feltmargin = styled.div`
+    margin-bottom: 2.5rem;
+`;
+
+const StyledFamilieDatovelger = styled(FamilieDatovelger)`
+    margin-right: 2rem;
+`;
+
+const StyledFerdigKnapp = styled(Knapp)`
+    margin-right: 0.5rem;
+`;
+
+const StyledFamilieTextarea = styled(FamilieTextarea)`
+    min-height: 8rem;
+`;
+
+interface IEndreUtbetaingsperiodeSkjemaProps {
+    åpenBehandling: IBehandling;
+    avbrytEndringAvUtbetalingsperiode: () => void;
+}
+
+const EndreUtbetaingsperiodeSkjema: React.FunctionComponent<IEndreUtbetaingsperiodeSkjemaProps> = ({
+    åpenBehandling,
+    avbrytEndringAvUtbetalingsperiode,
+}) => {
+    const { skjema } = useTidslinje();
```

Comment:
Bør vi lage en egen hook for disse periodene for å skille kompleksitet og funksjonelle behov?